[ACR] Improve instructions for authenticating a client (#21698)
Azure/azure-sdk-for-net: sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

### Prerequisites

You will need an [Azure subscription][azure_sub] and a [Container Registry service instance][container_registry_docs] for your application to connect to.

To create a new Container Registry, you can use the [Azure Portal][container_registry_create_portal],
[Azure PowerShell][container_registry_create_ps], or the [Azure CLI][container_registry_create_cli].
Here'a an example of creating a new registry using the Azure CLI:

```Powershell
az acr create --name myregistry --resource-group myresourcegroup --location westus --sku Basic
```

### Authenticate the client

> For your application to connect to your registry, you'll need to create a `ContainerRegistryClient` that can authenticate with the registry service.  The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.  

Reviewer: ```suggestion
For your application to connect to your registry, you'll need to create a `ContainerRegistryClient` that can authenticate with the registry service.  The [Azure Identity library][identity] makes it easy to add Azure Active Directory support for authentication.  
```

## Examples

> The following snippets show brief samples of common operations a user might use the ACR SDK library for.  

Reviewer: ```suggestion
The following snippets show brief samples of common developer scenarios using the ACR SDK library.  
```
